### PR TITLE
fix(slider): use a unique id for the helper text & use it by `aria-controls`

### PR DESCRIPTION
**Pull request by Kiarokh**
## Review:
- [ ] Commits are [atomic](https://seesparkbox.com/foundry/atomic_commits_with_git)
- [ ] Commits have the correct *type* for the changes made
- [ ] Commits with *breaking changes* are marked as such

### Browsers tested:
(Check any that applies, it's ok to leave boxes unchecked if testing something didn't seem relevant.)

Windows:
- [ ] Chrome
- [ ] Edge
- [ ] Firefox

Linux:
- [ ] Chrome
- [ ] Firefox

macOS:
- [ ] Chrome
- [ ] Firefox
- [ ] Safari

Mobile:
- [ ] Chrome on Android
- [ ] iOS

### CODE REVIEW

File: src/components/slider/slider.tsx [hunk 1]
```diff
@@ -101,13 +103,17 @@ export class Slider {
     private rootElement: HTMLLimelSliderElement;
 
     private mdcSlider: MDCSlider;
+    private labelId: string;
+    private helperTextId: string;
```
Comment (Kiarokh) [inline src/components/slider/slider.tsx:107]: or is it better to write these like this: `private helperTextId: string = createRandomString();` and get rid of stuff in the `constructor()` below 👇 ?

Comment (Befkadu1) [reply src/components/slider/slider.tsx:107]: It is fine to write it in the constructor.

**Review by Befkadu1 — APPROVED** [review]

**Comment by Befkadu1** [comment]
I don't think this PR should wait for this [PR](https://github.com/Lundalogik/lime-elements/pull/2379/commits/af9b1938107a6523412b4176d88ae23edde47539) to be merged first. What do you think @LawrenceBorst ?

**Review by Befkadu1 — CHANGES_REQUESTED** [review]
The reason I requested merging is not to merge the PR since I already approved it. Don't merge until @LawrenceBorst answers.

**Comment by LawrenceBorst** [comment]
> I don't think this PR should wait for this [PR](https://github.com/Lundalogik/lime-elements/pull/2379/commits/af9b1938107a6523412b4176d88ae23edde47539) to be merged first. What do you think @LawrenceBorst ?

In this case it doesn't seem like the other PR depends on this one. In any case, the other PR will be merged very soon

**Review by LawrenceBorst — APPROVED** [review]

**Review by Befkadu1 — APPROVED** [review]

**Pull request merged**